build-farm/make-adopt-build-farm.sh

```diff
@@ -132,6 +132,11 @@ then
     # OpenJ9 only supports building jdk-11 with jdk-11
     JDK_BOOT_VERSION="11"
   fi
+  if [ "${JAVA_FEATURE_VERSION}" == "19" ]; then
+    # To support reproducible-builds the jar/jmod --date option is required
+    # which is only available from jdk-19
+    JDK_BOOT_VERSION="19"
```

Comment:
Maybe future proof this as well?

Reply:
It's only building jdk-19 that we can't use N-1 as jdk-18 doesn't have the reproducible build options for jar&jmod.
Building jdk-20+ is fine, as jdk-19 does.